libs/blocks/section-metadata/section-metadata.css

```diff
@@ -9,7 +9,7 @@
 
 .section.dark {
   background-color: rgb(29 29 29);
-  color: rgb(235 235 235);
+  color: rgb(255 255 255);
```

Comment:
Not sure if you want to tie into it but you could also use `var(--background-color);`